config/locales/no/shared.yml

```diff
@@ -0,0 +1,3 @@
+"no":
```

Comment:
Is there a reason this key is in quotes?

Reply:
`no` is [treated as a boolean rather than a string](https://juliankniephoff.wordpress.com/2012/09/01/reserved-words-in-yaml-and-translating-booleans-in-rails/), so we get an error when the tests runs

Reply:
Interesting, thanks for explaining!